app/assets/src/components/AMRView.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import ReactTable from "react-table";
+import "react-table/react-table.css";
+import _ from "lodash";
+
+const columns = [
+  {
+    Header: "Antibiotic Class",
+    accessor: "drug_family"
+  },
+  {
+    Header: "Gene",
+    accessor: "gene",
+    aggregate: vals => _.size(vals)
+  },
+  {
+    Header: "Gene Family",
+    accessor: "gene_family",
+    aggregate: vals => _.size(vals)
+  },
+  {
+    // TODO: Make average for gene family level
+    Header: "Coverage",
+    accessor: "coverage",
+    aggregate: vals => _.sum(vals),
+    filterMethod: (filter, row) => row[filter.id] >= filter.value
+  },
+  {
+    Header: "Depth",
+    accessor: "depth",
+    aggregate: vals => _.sum(vals),
+    filterMethod: (filter, row) => row[filter.id] >= filter.value
+  }
+];
+
+class AMRView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { data: props.amr ? props.amr : [] };
```

Comment:
Why is the data put in `state`? Is it a requirement of ReactTable? Normally `state` is used for variables that will change during the session. The AMR data on the other hand seems static.

Reply:
That's true, I can look into it more and get back to you.

Reply:
As recommended by @cdebourcy, you should move the data out of state as you are not using it to influence any local elements.
This could even be a functional component.

Reply:
So for now, just literally change `this.state` to `this.data` or whatever

Reply:
Sounds good, resolved